<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>191</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="HW1Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW2Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW3Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW4Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW5Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW6Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW7Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW8Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="HW9Slider">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>61</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSpinBox" name="HW1Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW2Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW3Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW4Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW5Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW6Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW7Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW8Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="HW9Spin">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>57</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>HW1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>HW2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>HW3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>HW4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>HW5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>HW6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>HW7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>HW8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>HW9</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>211</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Midterm1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Midterm2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Courses: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Pic10C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pic10B</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="SchemaA">
       <property name="text">
        <string>Schema A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SchemaB">
       <property name="text">
        <string>Schema B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="final_grade">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Grade: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>HW1Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW1Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW1Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW1Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW2Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW2Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW3Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW3Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW2Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW2Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>205</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW3Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW3Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW4Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW4Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW4Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW4Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW5Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW5Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW5Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW5Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW6Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW6Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW6Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW6Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW7Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW7Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW7Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW7Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW8Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW8Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW8Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW8Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>359</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW9Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW9Spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HW9Spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HW9Slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
